### DISCUSSION TITLE
What is ALVR's MIT License plan?

### DISCUSSION THREAD

**Discussion by JeongJunSik** [alvr-org/ALVR#1630] [Q&A]
As far as I know, ALVR project is MIT license.

However, some files contain other license files.
For example, VrAPI_Helpers.h (Copyright   :   Copyright (c) Facebook Technologies, LLC and its affiliates. All rights reserved.)

Do you have any plans to remove these licenses?

**Comment by zmerp** [comment]
The SDKs are free to be used in opensource and closed source projects. I haven't checked but there should be no license conflict.
